### PR TITLE
Edit introductory section of user guide

### PR DESCRIPTION
**Pull request by SherwynNg**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -6,16 +6,78 @@
...
+InternBook isn't just another app - it's your personal assistant for managing internship applications effortlessly. With 
+InternBook, you can record crucial details like company names, application deadlines, contact emails, and the roles you
+are applying for, all in one organised place. It is optimised for a _Command line Interface_ (CLI), but you can use the
+advantages of a _Graphical User Interface_ (GUI). No more frantic searches through endless spreadsheets or sticky notes!
+
+### Why Choose InternBook?
+- **Simplicity:** Designed for students, whether you have a technical background or not. InternBook is easy to use and follow, 
+ensuring a smooth experience for everyone. Say goodbye to complicated Excel templates. InternBook offers a clean and 
+intuitive designed with your needs in mind.
+- **Organisation:** Keep your internship applications neat and tidy. No more digging through clutter - everything you
```
Comment (jasperng-nus) [inline docs/UserGuide.md:29]: I think this could be cluttered data or maybe clutters

File: docs/UserGuide.md [hunk 2]
```diff
@@ -6,16 +6,78 @@
...
+- **Reminder Functionality:** Set reminders for upcoming application deadlines, ensuring timely submissions and proactive
+management of internship opportunities.
+- **Sorting Options:**
+    - **Alphabetical**: Arrange applications alphabetically by company name for easy reference and quick navigation.
+    - **Start Date**: Sort applications based on the start date of the internship application, allowing users to prioritise 
+    applications based on upcoming opportunities.
+    - **End Date**: Sort applications according to the end date of the internship application, facilitating planning and 
+    scheduling by highlighting approaching deadlines.
+- **User-Friendly Interface:** With a clean and intuitive design, InternBook offers ease of navigation and accessibility,
+prioritising ser experience and satisfaction.
```
Comment (jasperng-nus) [inline docs/UserGuide.md:77]: I think there is typo, i think you meant "user"

File: docs/UserGuide.md [hunk 3]
```diff
@@ -52,7 +114,16 @@ It is optimized for a _Command Line Interface_ (CLI), but you can still use the
 
 5. Refer to the [Features](#features) below for details of each command.
 
-
+--------------------------------------------------------------------------------------------------------------------
+## How to use the guide
+- The guide introduces the features of our application and shows you an example on the different commands. Stating which
+are the compulsory fields, in a particular format and constraints to it.
+- After the features section, it consists of the frequency asked questions (FAQ), so if you have any doubts, you might
```
Comment (jasperng-nus) [inline docs/UserGuide.md:121]: I think you meant "frequently" instead

**Review by jasperng-nus — COMMENTED** [review]
Overall LGTM, just a few minor spelling errors

File: docs/UserGuide.md [hunk 4]
```diff
@@ -6,16 +6,78 @@
...
 # InternBook User Guide
 
-Welcome to InternBook!
-
-InternBook is your go-to fast, reliable and customizable **desktop app for tracking internship details**
-
-It is optimized for a _Command Line Interface_ (CLI), but you can still use the advantages of a _Graphical User Interface_ (GUI).
+## Welcome to InternBook!
+Hey there, future interns! Since you are here, we summed that you are also a student, like us, who is starting your internship search. 
+Welcome to InternBook, your go-to companion in the exciting journey of applying for internships.
```
[the remark this answers is not in the document]
Comment (AjayArvind2207) [reply docs/UserGuide.md:11]: I agree with this

File: docs/UserGuide.md [hunk 5]
```diff
@@ -4,18 +4,80 @@
...
+### Who Are We?
+We are a dynamic team of five students who have been in your shoes, feeling the struggle of juggling multiple internship
+applications while trying to keep track of every detail. We get it - those Excel spreadsheets might seem like a good idea 
+at first, but let's face it, they quickly become a cluttered mess, making it harder to find what you need and leading to 
+inconsistencies in your data. That's why we decided to roll up our sleeves and create InternBook!
 
-It is optimized for a _Command Line Interface_ (CLI), but you can still use the advantages of a _Graphical User Interface_ (GUI).
+### What is InternBook?
+InternBook isn't just another app - it's your personal assistant for managing internship applications effortlessly. With 
+InternBook, you can record crucial details like company names, application deadlines, contact emails, and the roles you
```
Comment (AjayArvind2207) [inline docs/UserGuide.md:21]: These things we can put in pointers

File: docs/UserGuide.md [hunk 6]
```diff
@@ -4,18 +4,80 @@
...
+are applying for, all in one organised place. It is optimised for a _Command line Interface_ (CLI), but you can use the
+advantages of a _Graphical User Interface_ (GUI). No more frantic searches through endless spreadsheets or sticky notes!
+
+### Why Choose InternBook?
+- **Simplicity:** Designed for students, whether you have a technical background or not. InternBook is easy to use and follow, 
+ensuring a smooth experience for everyone. Say goodbye to complicated Excel templates. InternBook offers a clean and 
+intuitive designed with your needs in mind.
+- **Organisation:** Keep your internship applications neat and tidy. No more digging through clutter - everything you 
+need is right at your fingertips.
+- **Consistency:** Say hello to reliable data. With InternBook, there are error checking for inputs to ensure that the
```
Comment (AjayArvind2207) [inline docs/UserGuide.md:31]:
```suggestion
- **Consistency:** Say hello to reliable data. With InternBook, there is error checking for inputs to ensure that the 
```

File: docs/UserGuide.md [hunk 7]
```diff
@@ -52,7 +114,16 @@ It is optimized for a _Command Line Interface_ (CLI), but you can still use the
 
 5. Refer to the [Features](#features) below for details of each command.
 
-
+--------------------------------------------------------------------------------------------------------------------
+## How to use the guide
+- The guide introduces the features of our application and shows you an example on the different commands. Stating which
+are the compulsory fields, in a particular format and constraints to it.
```
Comment (AjayArvind2207) [inline docs/UserGuide.md:120]:
```suggestion
- The guide introduces the features of our application and shows you examples of the different commands in action. It also provides the field constraints, including permitted values and compulsory or optional parameters.
```

File: docs/UserGuide.md (hunk 3 above)
Comment (AjayArvind2207) [inline docs/UserGuide.md:121]:
```suggestion
- After the features section, we have a frequently asked questions (FAQ) section, so if you have any doubts, you might
```

File: docs/UserGuide.md [hunk 8]
```diff
@@ -52,7 +114,16 @@ It is optimized for a _Command Line Interface_ (CLI), but you can still use the
...
+- After the features section, it consists of the frequency asked questions (FAQ), so if you have any doubts, you might
+want to refer to the FAQ to see whether your questions can be answered.
+- After the FAQ section, it consists of known issues, which may contain bugs that we, the developers, are working on.
```
Comment (AjayArvind2207) [inline docs/UserGuide.md:123]:
```suggestion
- After the FAQ section, we have an issue tracker which may contain bugs that we, the developers, are working on. 
```

File: docs/UserGuide.md [hunk 9]
```diff
@@ -52,7 +114,16 @@ It is optimized for a _Command Line Interface_ (CLI), but you can still use the
...
-
+--------------------------------------------------------------------------------------------------------------------
+## How to use the guide
+- The guide introduces the features of our application and shows you an example on the different commands. Stating which
+are the compulsory fields, in a particular format and constraints to it.
+- After the features section, it consists of the frequency asked questions (FAQ), so if you have any doubts, you might
+want to refer to the FAQ to see whether your questions can be answered.
+- After the FAQ section, it consists of known issues, which may contain bugs that we, the developers, are working on. 
+Please be patient with us while we fix the bugs!
+- The last section consist of the command summary where a summary for the commands are listed in a table for better and
```
Comment (AjayArvind2207) [inline docs/UserGuide.md:125]:
```suggestion
- The last section consists of the command summary where a summary for every command is listed in a table for better and 
```

**Review by AjayArvind2207 — COMMENTED** [review]
LGTM on the whole! Just some minor nits relating to grammar and sentence structure.

**Comment by AjayArvind2207** [comment]
LGTM 👍

**Review by jasperng-nus — COMMENTED** [review]
LGTM

**Pull request merged**